pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"net/http"
 
+	appsapiv1beta1 "k8s.io/api/apps/v1beta1"
```

Comment:
It's pretty weird to have both "appsv1beta1" (internal version) and this (external) in the list, but it seems necessary. I guess it's needed because ordinarily the registry files work 100% in the internal version. If you were to make a change I'd maybe call this externalappsv1beta1, maybe, just for clarity.